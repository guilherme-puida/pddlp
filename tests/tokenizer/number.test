0
10
1.0
12.923
10 11 12
