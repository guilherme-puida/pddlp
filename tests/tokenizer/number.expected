[01:01] 0
[02:01] 10
[03:01] 1.0
[04:01] 12.923
[05:01] 10
[05:04] 11
[05:07] 12
EOF
